hazelcast/src/main/java/com/hazelcast/map/impl/wan/WanMapEntryView.java

```diff
@@ -135,6 +138,10 @@ public void writeData(ObjectDataOutput out) throws IOException {
         out.writeLong(lastUpdateTime);
         out.writeLong(version);
         out.writeLong(ttl);
+        out.writeBoolean(maxIdle != null);
```

Comment:
I think the original reason why it was boxed was that we couldn't add it in WAN in 3.x and we needed to distinguish between the case where `maxIdle` was unknown (null) and when it was the default (-1). Now, seems to me that all `EntryView` implementations have a `maxIdle`. Can you double-check and see if we can finally remove this boxing?

Reply:
Done